src/openmc_driver.cpp

```diff
@@ -21,6 +20,31 @@ OpenmcDriver::OpenmcDriver(MPI_Comm comm)
     err_chk(openmc_init(0, nullptr, &comm));
   }
   MPI_Barrier(MPI_COMM_WORLD);
+
+  // determine number of fissionable cells in model to aid in catching
+  // improperly mapped problems
+  n_fissionable_cells_ = 0;
+  for (int i = 0; i < cells_size(); ++i) {
+    int type;
+    int32_t* indices;
+    int32_t n;
+    err_chk(openmc_cell_get_fill(i, &type, &indices, &n));
+
+    // only check for cells filled with type FILL_MATERIAL (evaluated to '1' enum)
+    if (type == 1) {
+      for (int j = 0; j < n; ++j) {
```

Comment:
This isn't a narrowing conversion (since `n` is an `int32_t`, not some size type), so it's correct.  But might want to consider using `gsl::index` anyway.